### ISSUE TITLE
[Security] More secure passwords

### ISSUE THREAD

**Issue by realjackasterisk** [sp614x/optifine#3794]
OptiFine passwords are incredibly simple right now. And I shouldn't have to say that's not good.

Currently, OptiFine passwords look like this:
- Always exactly 10 characters in length
- Use letters
- Use numbers

....And that's it. 
But it's randomly generated, so that should be okay, right...?

For your consideration: This is my *actual* optifine.net password.
![image](https://user-images.githubusercontent.com/31686191/80302057-9f71d580-875c-11ea-86ef-7e972c2e75bb.png)

To improve this, I would suggest generating passwords with these parameters:
- Random length, between 12-24 characters.
- Use letters, numbers, AND special characters (`!#$%^&*()_+`)

**Comment by Andrew-J-Larson** [comment]
I agree... getting tired of 1password listing this as my only weak password too...
